jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/ConfigsTest.java

```diff
@@ -42,109 +48,99 @@ public enum ConfigWithImplementations implements Configs.Config {
   }
 
   @Test
-  public void configInterfaceWithDefaults() {
+  void configInterfaceWithDefaults() {
     assertThat(ConfigWithDefaults.ONE.getKey()).isEqualTo("ONE");
     assertThat(ConfigWithDefaults.ONE.getDefaultValue()).isNull();
     assertThat(ConfigWithDefaults.TWO.getKey()).isEqualTo("TWO");
     assertThat(ConfigWithDefaults.TWO.getDefaultValue()).isNull();
   }
 
   @Test
-  public void configInterfaceWithImplementations() {
+  void configInterfaceWithImplementations() {
     assertThat(ConfigWithImplementations.ONE.getKey()).isEqualTo("one");
     assertThat(ConfigWithImplementations.ONE.getDefaultValue()).isEqualTo("this is the default value one");
     assertThat(ConfigWithImplementations.TWO.getKey()).isEqualTo("two");
     assertThat(ConfigWithImplementations.TWO.getDefaultValue()).isEqualTo("default for two");
   }
 
   @Test
-  public void asIntValueWithValidStringShouldReturnParsed() {
+  void asIntValueWithValidStringShouldReturnParsed() {
     // When
     final int result = Configs.asInt("2");
     // Then
     assertThat(result).isEqualTo(2);
   }
 
-  @Test(expected = NumberFormatException.class)
-  public void asIntValueWithInvalidStringShouldThrowException() {
-    // When
-    Configs.asInt("2.15");
-    // Then
-    fail();
+  @Test
+  void asIntValueWithInvalidStringShouldThrowException() {
+    assertThrows(NumberFormatException.class, () -> {
+      // When
+      Configs.asInt("2.15");
+      // Then
+      fail();
```

Comment:
I think `fail()` shouldn't be required, right? Could this be simplified like this?

```java
assertThrows(NumberFormatException.class, () -> Configs.asInt("2.15"));
```